src/codeu/chat/client/simplegui/ChatSimpleGui.java

```diff
@@ -41,6 +41,14 @@ public ChatSimpleGui(Controller controller, View view) {
         clientContext = new ClientContext(controller, view);
     }
 
+    static public void hideFrame() {
+        mainFrame.setVisible(false);
```

Comment:
Doing this is kind of dangerous. Because this is static, it's very easy to call hideFrame() before mainFrame was ever set, which will raise a NullPointerException.  If you're doing this, check if mainFrame == null before doing .setVisible